packages/vite-plugin-angular/esbuild.ts

```diff
@@ -1,12 +1,17 @@
 import analog from '@analogjs/vite-plugin-angular';
 import type * as esbuild from 'esbuild';
+import { PluginOptions } from './src';
 
-export const analogSFC: () => esbuild.Plugin = () => ({
+export const analogSFC: (options?: PluginOptions) => esbuild.Plugin = (
+  options
+) => ({
   name: 'analog-sfc-esbuild-plugin',
   async setup(build) {
-    const analogPlugins: any = analog({
-      experimental: { supportAnalogFormat: true },
-    });
+    const analogPlugins: any = analog(
+      options ?? {
+        experimental: { supportAnalogFormat: true },
+      }
+    );
```

Comment:
I think the flag should still be included by default

```suggestion
    const analogPlugins: any = analog({
       ...(options || {}),
        experimental: {
          supportAnalogFormat: true,
          ...(options?.experimental || {})
        },
    });
```